components/automate-ui/e2e/event-feed-page.e2e-spec.ts

```diff
@@ -9,9 +9,18 @@ describe('Event Feed: Guitar String Graph Interaction', () => {
 
   it('should have a working dropdown filter', () => {
     const select = element(by.css('app-event-feed-select chef-select'));
-    select.click();
 
-    expect(select.getAttribute('class')).toContain('active');
+    // Click the event feed dropdown filter and wait until the filter turns 'active'
+    select.click().then(() => {
+      return new Promise(function(resolve) {
+        browser.wait(() => {
+          return select.getAttribute('class').then((values: string) => {
+            return values.indexOf('active') > 0;
```

Comment:
Probably OK, but note that this would be happy with a class name of `inactive` also. Ideally should separate the class by words, e.g., `values.split(' ').includes('active')`